setuptools/command/sdist.py

```diff
@@ -200,22 +200,32 @@ def read_manifest(self):
         manifest.close()
 
     def check_license(self):
-        """Checks if license_file' is configured and adds it to
-        'self.filelist' if the value contains a valid path.
+        """Checks if license_file' or 'license_files' is configured and adds any
+        valid paths to 'self.filelist'.
         """
 
+        files = set()
```

Comment:
Use of `set` will lead to non-deterministic builds for projects with more than one license. You'll want to use `orderedset` or a `dict` (ignoring key values) or a `list` (and filter duplicates).

Reply:
Fixed